tasks/10.test.js

```diff
@@ -0,0 +1,24 @@
+const { CalculatorPr, Calculator } = require("./10");
+
+describe("Calculator", () => {
+  describe("Prototype", () => {
+    test("Calculate values", () => {
+      var calcPr = new CalculatorPr(0);
+      const result = calcPr.add(5).multiply(2).add(20).divide(3).result;
+
+      const expected = 10;
```

Comment:
Avoid mixing variable types -- use modern ones ;) let, const

Reply:
Good point. Copy paste mistake left. Will avoid that.